ProjectLighthouse/Helpers/VersionHelper.cs

```diff
@@ -48,13 +51,16 @@ static VersionHelper()
                 "Please make sure you are properly disclosing the source code to any users who may be using this instance.",
                 LogArea.Startup
             );
+            Revision = $"{Branch}-dirty_r{rawRevision}";
             CanCheckForUpdates = false;
         }
     }
 
     public static string CommitHash { get; set; }
     public static string Branch { get; set; }
-    public static string FullVersion => $"Project Lighthouse {Branch}@{CommitHash} {Build} ({ServerConfiguration.Instance.Customization.ServerName})";
+    public static string Revision { get; set; }
+    public static string EnvVer => $"{ServerConfiguration.Instance.Customization.EnvironmentName} {Revision}";
```

Comment:
This variable name is a little unclear...

Perhaps renaming it to something like `ShortVersion`/`BrandVersion` or adding some XMLDoc could help?

Reply:
is the way it currently is fine?

Reply:
XMLDoc example:
```csharp
/// <summary>
/// The server's branding to show to LBP clients. Shows the environment name next to the revision.
/// </summary>
 public static string BrandVersion => "";
```

That way, you can hover over a reference to that variable and it will show you the summary.

This is in opposition to a comment; where you first have to know that it's there and you have to go to the class to look at it.

Reply:
hold on which variable are you talking about exactly?

it's not telling me exactly which line you're talking about

Reply:
`EnvVer`.

Reply:
documented both